src/Lightning/Node/Api.hs

```diff
@@ -19,7 +19,7 @@ import Servant.Auth (Auth)
 
 import Authorization.Macaroon (Macaroon)
 import Lightning.Node.Api.GetInfo as A (Address (..), NodeInfo (..))
-import Lightning.Node.Api.Invoice as A (InvoiceReq (..), InvoiceRep (..))
+import Lightning.Node.Api.Invoice as A (InvoiceReq (..), InvoiceLabel(..), InvoiceRep (..))
```

Comment:
This sorting order is strange. Strictly speaking it should be alphabetical, but it was already mixed up before :).